frontend/app/src/services/defi/defi-api.ts

```diff
@@ -127,4 +127,9 @@ export class DefiApi {
     const url = 'blockchains/ETH/modules/sushiswap/history/events';
     return fetchExternalAsync(this.api, url);
   }
+
+  async fetchPickleBalances(): Promise<PendingTask> {
```

Comment:
At least from a quick glance, this method seems to be unused at the moment.
We need to implement an action that would retrieve the data.

If we want to show them under the Deposits/Liabilities section at least.

Reply:
> At least from a quick glance, this method seems to be unused at the moment.
> We need to implement an action that would retrieve the data.
> 
> If we want to show them under the Deposits/Liabilities section or inject them in the frontend overview at least.

Yes it's my bad. It shouldn't be needed. The only thing that the PR does is to add the pickle-finance module. The backend, when the module is activated, takes care of querying the balances and update them.

This comes because I wanted to updated the DeFi card but as we discussed is something that can be done in another PR when we add more features to this protocol